test/src/test_derive.rs

```diff
@@ -322,3 +323,66 @@ fn test_derive_nativeclass_without_constructor() -> bool {
 
     ok
 }
+
+fn test_derive_nativeclass_with_property_get_set() -> bool {
+    #[derive(NativeClass)]
+    #[inherit(Node)]
+    struct CustomGetSet {
+        pub get_called: Cell<i32>,
+        pub set_called: Cell<i32>,
+        #[property(get = "Self::get_foo", set = "Self::set_foo")]
+        pub foo: i32,
+    }
+    #[methods]
+    impl CustomGetSet {
+        fn new(_onwer: &Node) -> Self {
+            Self {
+                get_called: Cell::new(0),
+                set_called: Cell::new(0),
+                foo: 0,
+            }
+        }
+        fn get_foo<'a>(&'a self, _owner: TRef<Node>) -> &'a i32 {
```

Comment:
This lifetime can be inferred, no? There's no other reference parameter than `self`.

```rs
fn get_foo(&self, _owner: TRef<Node>) -> &i32 {
```

Reply:
It's a very subtle difference but `TRef` does have an elided lifetime parameter here: `TRef<Node>` = `TRef<'_, Node>`.